Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+                                    deleteWalkthroughCookies();
+                                    Shepherd.activeTour.back();
+                                    return;
+                                }
+
+                                addShepherdQueryParams();
+                                setWalkthroughCookies(this.tour.options.id, 'deployment_manage_steps');
+                            },
+                        },
+                    },
+                    {
+                        title: 'Import/export, deployment, <br> deployment plan',
+                        text: 'Click on the <i>"Add Step"</i> button.',
+                        attachTo: { element: '.btn.btn-primary.btn-sm', on: 'top' },
+                        buttons: [
+                            {
+                                action: function () {
+                                    goToRelativePage(
+                                        Shepherd.activeTour.options.id,
+                                        'deployment_deployment_plan_published',
+                                        'Admin',
+                                        'Admin/DeploymentPlan/Index');
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                        ],
+                        id: 'deployment_manage_steps',
+                        advanceOn: { selector: '.btn.btn-primary.btn-sm', event: 'click' },
+                    },
+                    {
+                        title: 'Import/export, deployment, <br> deployment plan',
+                        text: 'Here you can see all the steps that you can add.',
+                        attachTo: { element: '.modal-body', on: 'top' },
+                        canClickTarget: false,
+                        buttons: [
+                            {
+                                action: function () {
+                                    goToRelativePage(
+                                        Shepherd.activeTour.options.id,
+                                        'deployment_deployment_plan_published',
+                                        'Admin',
+                                        'Admin/DeploymentPlan/Index');
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'deployment_manage_steps',
+                        advanceOn: { selector: '.btn.btn-primary.btn-sm', event: 'click' },
+                    },
+                    {
+                        title: 'Import/export, deployment, <br> deployment plan',
+                        text: 'Let\'s filter for <i>"Update Content Definitions"</i>.',
```

Comment:
When you click on the next please fill the filter with Update Content Definitions, otherwise it gets buggy.